src/agent/src/main.rs

```diff
@@ -346,7 +346,7 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
 
     if let Some(SubCommand::Init {}) = args.subcmd {
         reset_sigpipe();
-        rustjail::container::init_child();
+        rustjail::container::init_child(AGENT_CONFIG.expose_configfs);
```

Comment:
This is expected to crash if the agent isn't run as init because `/proc` isn't mounted yet at this point. This causes `from_cmdline()` to fail at https://github.com/kata-containers/kata-containers/blob/407df4328bd7983321fa50037dcf91d74fad6566/src/agent/src/config.rs#L437 since `/proc/cmdline` doesn't exist. Then we hit the `unwrap()` at https://github.com/kata-containers/kata-containers/blob/407df4328bd7983321fa50037dcf91d74fad6566/src/agent/src/main.rs#L132.

Question is : how is it possible that CI doesn't catch that ?